packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -585,13 +851,64 @@ class RoundSliderThumbShape extends SliderComponentShape {
       end: sliderTheme.thumbColor,
     );
     canvas.drawCircle(
-      thumbCenter,
+      center,
       radiusTween.evaluate(enableAnimation),
       Paint()..color = colorTween.evaluate(enableAnimation),
     );
   }
 }
 
+/// This is the default shape to a [Slider]'s thumb overlay if no
+/// other shape is specified.
+///
+/// See also:
+///
+///  * [Slider] for the component that this is meant to display this shape.
+///  * [SliderThemeData] where an instance of this class is set to inform the
+///    slider of the shape of the its thumb.
+class RoundSliderOverlayShape extends SliderComponentShape {
+  /// Create a slider thumb overlay that draws a circle.
+  const RoundSliderOverlayShape();
+
+  static const double _overlayRadius = 16.0;
+
+  @override
+  Size getPreferredSize(bool isEnabled, bool isDiscrete) {
+    return const Size.fromRadius(_overlayRadius);
+  }
+
+  @override
+  void paint(
+    PaintingContext context,
+    Offset center, {
+    Animation<double> activationAnimation,
+    Animation<double> enableAnimation,
+    bool isDiscrete,
+    TextPainter labelPainter,
+    RenderBox parentBox,
+    SliderThemeData sliderTheme,
+    TextDirection textDirection,
+    double value,
+    }) {
```

Comment:
I think this `}) {` should be two spaces to the left, to match the `void paint(`

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#if-you-have-a-newline-after-some-opening-punctuation-match-it-on-the-closing-punctuation

Reply:
fixed, and indentation beneath